src/runtime/Makefile

```diff
@@ -167,6 +167,10 @@ DEFDISABLEGUESTEMPTYDIR := false
 DEFAULTEXPFEATURES := []
 
 DEFDISABLESELINUX := false
+#Default SeccomSandbox param
+#The same default policy is used by libvirt
+#More explanation on https://lists.gnu.org/archive/html/qemu-devel/2017-02/msg03348.html
+DEFSECCOMPSANDBOXPARAM := on,obsolete=deny,elevateprivileges=deny,spawn=deny,resourcecontrol=deny
```

Comment:
I would need to test how `resourcecontrol=deny` especially is affecting GPU or NIC passthrough.

Reply:
All in all, I am in favor of adding this, the more of an attack surface we can minimize we should do it. Would be interesting if we could extend the whitelist even more for Kata.

Reply:
I don't think we should change the existing defaults -- if we want to do this, it should be done at 2.0 timeframe. We can have a suggested value in comments, but default should keep existing behavior (I think that default would be "" ? )

Reply:
@egernst The default is disabled. In the configuration-qemu.toml.in file, the configuration is commented out.